CHANGELOG.md

```diff
@@ -4,18 +4,23 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
-### Changed
+### Fixed
```

Comment:
I assume removing the entry for the Rust dependency bumps was intentional?

Reply:
Jep, as it should not be user-facing